src/components/FormNewTask/FormNewTask.stories.tsx

```diff
@@ -12,4 +12,10 @@ export default {
   decorators: [decoratorCentered],
 }
 
-export const example = () => <FormNewTask />
+export const example = () => (
+  <FormNewTask
+    onAddTask={(text) => {
```

Comment:
⛏️ This is where you could do that `action` thing from storybook. If you're in storybook, whatever string you pass the `action` will log to the console in storybook when that function is executed.

`import { action } from '@storybook/addon-actions';`

`onTaskAdd={action('onTaskAdd')}`

Reply:
done on a different story in ADASP-25-modal (ButtonCreateTask.stories.tsx)